lib/tasks/dev.rake

```diff
@@ -0,0 +1,77 @@
+
+task sample_data: :environment do
+  p "creating sample data"
+  starting = Time.now
+
+  Member.delete_all
+  Contact.delete_all
+  Group.delete_all
+  User.delete_all
+
+  5.times do
+    number = rand.to_s[2..11] 
+    name = Faker::Name.first_name
+    User.create(
+      email: "#{name}@example.com",
+      password: "password",
+      username: name.downcase,
```

Comment:
You have a uniqueness validation on username

https://github.com/jrohkohl/contactify/blob/db73b80ca9ca9d3787d4cb0eb859e201459dc499/app/models/user.rb#L27

In this `5.times` loop, each user is using the same username, so only one will save.

You could do something like

```suggestion
    User.create(
      email: "#{name}@example.com",
      password: "password",
      username: Faker::Name.first_name.downcase,
```

That way, the username is always different.